helix-term/src/main.rs

```diff
@@ -91,8 +92,39 @@ FLAGS:
 
     setup_logging(logpath, args.verbosity).context("failed to initialize logging")?;
 
+    let themes = std::sync::Arc::new(Themes::load());
```

Comment:
This should really be delayed, we don't want to load all available themes into memory. `list-themes` should read the directory entries (I think we should move `contrib/themes` under `runtime/themes`), then when setting the theme we actually read the file.